app/waf/src/controllers/application.controller.ts

```diff
@@ -62,7 +59,7 @@ export class ApplicationController extends BaseController {
     public declarationRepository: DeclarationRepository,
     @repository(AdcRepository)
     public adcRepository: AdcRepository,
-    @inject('services.AS3Service') public as3Service: AS3Service,
+    @inject('services.ASGService') public asgService: ASGService,
```

Comment:
The `asgService` in this class is not used, can be removed.

Reply:
it's actually used in   tests . so i modified tests code to get rid of it.

Reply:
good.